native/src/routes/Landing.tsx

```diff
@@ -51,17 +47,9 @@ const Landing = ({
     <>
       <Wrapper {...testID('Landing-Page')}>
         <Heading clearResourcesAndCache={clearResourcesAndCache} theme={theme} />
-        <FilterableCitySelector
-          theme={theme}
-          cities={cities}
-          t={t}
-          locationInformation={locationInformation}
-          navigateToDashboard={navigateTo}
-        />
+        <CitySelector cities={cities} t={t} navigateToDashboard={navigateTo} />
       </Wrapper>
-      {buildConfig().featureFlags.cityNotCooperating && (
-        <CityNotCooperatingFooter navigateToCityNotCooperating={navigateToCityNotCooperating} theme={theme} />
-      )}
+      <CityNotCooperatingFooter navigateToCityNotCooperating={navigateToCityNotCooperating} theme={theme} />
```

Comment:
Why have you removed the check for the featureFlag. Is the cityNotCooperating Footer now shown in Malte?

Reply:
There is a feature flag check in `CityNotCooperatingFooter` as well returning null if it is false. No way to check in both places.